genologics/entities.py

```diff
@@ -298,15 +298,19 @@ def post(self):
         self.lims.post(self.uri, data)
 
     @classmethod
-    def _create(cls, lims, creation_tag=None, **kwargs):
+    def _create(cls, lims, creation_tag=None, udfs=None, **kwargs):
         """Create an instance from attributes and return it"""
+        if not udfs:
+            udfs={}
```

Comment:
Can't you just give `udfs={}` above instead of `udfs=None` while defining *parameters*

Reply:
No, because on what that implies : 
```>>> class T(object):
...   def mymethod(self, a, b={}):
...     b[a]='test'
...     print b
... 
>>> k=T()
>>> k.mymethod(a='test')
{'test': 'test'}
>>> k.mymethod(a='test2')
{'test': 'test', 'test2': 'test'}
>>> l=T()
>>> l.mymethod('test3')
{'test': 'test', 'test3': 'test', 'test2': 'test'}
```
You never want to use mutables as method defaults.

Reply:
Neat @Galithil 👏 Well explained the disaster it could make 😅